### ISSUE TITLE
Variable Background Colors

### ISSUE THREAD

**Issue by irmerk** [accordproject/markdown-editor#126]
**Is your feature request related to a problem? Please describe.**
Variables should have customizable background colors, as well as only having colors in `lockText` mode.

**Describe the solution you'd like**
Allow for props to dictate the color of the variables, and only display this when `lockText` is true.

**Describe alternatives you've considered**
Open to input.

**Additional context**
Referenced in #124

**Comment by gsilvamartin** [comment]
Please assign this to me.

**Comment by irmerk** [comment]
Opening this back up after 6 days stale. @gsilvamartin let me know if you're still working on it and I'll reassign.

**Comment by mbtamuli** [comment]
@gsilvamartin are you still working on this?

@irmerk Can I take a shot at this?

**Comment by mbtamuli** [comment]
@irmerk I am a little confused by this issue.

>  Variables should have customizable background colors, as well as only having colors in lockText mode.
1. So here, does `Variables` mean the variables in the template as described here https://docs.accordproject.org/docs/spec-template.html#grammar?

2. I can see example of variable usage here - https://studio.accordproject.org/. How can I run this locally?

**Comment by irmerk** [comment]
The yellow highlighted portions in `cicero-ui` locally here:
<img width="939" alt="Screen Shot 2019-10-23 at 14 49 32 PM" src="https://user-images.githubusercontent.com/36460856/67424591-6b96d700-f5a4-11e9-9af2-ccdd880808eb.png">

These are generated from the [`markdown-transform`](https://github.com/accordproject/markdown-transform/), some parts to refer to:
- [`slateToCiceroMarkDom`](https://github.com/accordproject/markdown-transform/blob/master/packages/markdown-slate/src/slateToCiceroMarkDom.js#L58)
- [`ToSlateVisitor`](https://github.com/accordproject/markdown-transform/blob/master/packages/markdown-slate/src/ToSlateVisitor.js#L148)
- [`FromCiceroVisitor`](https://github.com/accordproject/markdown-transform/blob/master/packages/markdown-cicero/src/FromCiceroVisitor.js#L108)

Though that could potentially be overkill because I believe Slate is the one adding the yellow highlighting annotation, which will be in this repo. @dselman any thoughts?

You'll want to run this locally with `cicero-ui` linked through `npm link`

**Comment by irmerk** [comment]
@mbtamuli any progress on this?

**Comment by mbtamuli** [comment]
I wasn't able to work on this. You can unassign me for now.

**Comment by elit-altum** [comment]
Can I take a shot with this ?

**Comment by irmerk** [comment]
@elit-altum Yes but I believe this may need to be moved to `cicero-ui`, which is where variables are defined and used. I'll transfer it there and assign you.
